directdemod/merger/lib_checker.py

```diff
@@ -0,0 +1,88 @@
+'''
+library checker
```

Comment:
The lib_checker.py is not just limited to the merger. Similar stuff is also used in other files. Hence, to enhance code reuse, please move this to a common file in directdemod directory. misc.py would do.

Reply:
Should I just move it there or should I also find where these methods could be applied, and delete duplicate code?

Reply:
You can just move it for now, deleting duplicate code will be for later.